src/System.Binary.Base64/System.Binary.Base64.csproj

```diff
@@ -2,7 +2,7 @@
   <Import Project="..\..\tools\common.props" />
   <PropertyGroup>
     <Description>Non-allocating Base64 encoder and decoded</Description>
-    <TargetFramework>netstandard1.1</TargetFramework>
+    <TargetFramework>netstandard1.3</TargetFramework>
```

Comment:
Why are we bumping the version?

Reply:
This is from a separate PR. I'm working on that PR as a base instead of master.